### ISSUE TITLE
Images not rendered in PDF

### ISSUE THREAD

**Issue by Liriel** [splitbrain/dokuwiki-plugin-dw2pdf#145]
Whenever I export to PDF the images are not rendered (there's a little document icon with a red X instead)
   * dokuwiki 2014-05-05 "Ponder Stibbons" 
   * apache2 2.2.27-r4 
   * php 5.5
   * dw2pdf 2014-09-28
   * @all has no access rights
   * userewrite is set to "none"
   * authtype is set to "authad"

The image is referenced like this:
```
{{:MyImage.png?nolink|}}
```
already tried: 
   * disable ACL entirely
   * use plain auth
   * change @all permission to "updload"

**Comment by sandersjds** [comment]
I had the same problem with the latest versions of dw2pdf. The images were showing up in the produced PDFs as IE-esque boxes with red X's. I had multiple images in the document, and only those that had PNG alpha channels failed to show up.

Per your last recommendation, I enabled mPDF image debugging. I opened the mpdf config file (/var/www/dokuwiki/lib/plugins/dw2pdf/mpdf/config.php) and altered the appropriate line:

`$this->showImageErrors = false;`

...becomes: 

`$this->showImageErrors = true;`

I then attempted to export the PDF again and received this for my efforts:

> mPDF error: IMAGE Error (/var/www/dokuwiki/data/media/path/png-with-alpha.png): GD library required for PNG image (alpha channel)

My server is Ubuntu 14.04; to fix this I added GD support by running:

`sudo apt-get install php5-gd`
`apache2ctl graceful # or whatever method you prefer to restart apache`

Moral of the story: don't assume you've got GD just because *some* images were working. Mine works now; thank you for the hint!
